### PR TITLE
Task3_Module4

### PR DESCRIPTION
**Pull request by OlegMazurenko**

### CODE REVIEW

File: DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            _productService = productService;
+        }
+
+        [HttpGet]
+        public IActionResult Index()
+        {
+            return View("Index", _productService.GetProducts());
+        }
+
+        [HttpGet("details")]
```
Comment (yevhenbilushko) [inline DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs:27]: [HttpGet("details")]
->
[HttpGet]

cause method already has name "Details"

Comment (yevhenbilushko) [reply DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs:27]: do the same with others please

Comment (yevhenbilushko) [reply DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs:27]: remove name from attribute if it is the same with method name

**Review by yevhenbilushko — APPROVED** [review]
fix and merge

File: DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs (hunk 1 above)
Comment (OlegMazurenko) [reply DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs:27]: Decided to keep as is

**Pull request merged**